### PR TITLE
Visual Novel

### PR DESCRIPTION
**Pull request by ynks**

### CODE REVIEW

**Comment by ynks** [comment]
This pull request contains: 
- Visual Novel UI & system
- Custom made menu & button system
- Progress bar Popup
- Bullet Popup

It's a very heavy pull request sry, should have worked on two different branches

File: Assets/Core/UniversalRP.asset
Comment (dariormt) [inline Assets/Core/UniversalRP.asset:1]: Why this is changed? we aren't using 3D shadows nor lens flares we should probably look into this file change and revert it

**Review by dariormt — CHANGES_REQUESTED** [review]
The reviewing process wasn't as bad as I expected, We should look into UniversalRP file changes and address them before merging

File: Assets/Core/UniversalRP.asset
Comment (ynks) [reply Assets/Core/UniversalRP.asset:1]: Looking at the commit history for the file, it changed on [68ab503663a671825a8cfa233a89bc5d0c4ccd69](https://github.com/Nullptr-Studios/Project-Hotline/commit/68ab503663a671825a8cfa233a89bc5d0c4ccd69#diff-681ca7bef5091a5890e34c4980cdc33389ab81e4689b11da8bcee9db59697853), which was a merge from dev. I haven't changed anything directly myself

Comment (dariormt) [reply Assets/Core/UniversalRP.asset:1]: Ok, we'll leave it as it is.

**Review by dariormt — APPROVED** [review]

**Pull request merged**